ballista/rust/scheduler/src/api/mod.rs

```diff
@@ -85,7 +85,7 @@ fn with_data_server<T: AsLogicalPlan + Clone, U: 'static + AsExecutionPlan>(
 pub fn get_routes<T: AsLogicalPlan + Clone, U: 'static + AsExecutionPlan>(
     scheduler_server: SchedulerServer<T, U>,
 ) -> BoxedFilter<(impl Reply,)> {
-    let routes = warp::path("state")
+    let routes = warp::path!("api" / "state")
```

Comment:
overloading the `/` operator? Nothing to do with this PR, but not a fan after seeing abuse in C++ & C#.

Reply:
Agreed. I am just beginning to learn about the warp framework, so will see if this can be avoided as I learn more.